third_party/2and3/click/decorators.pyi

```diff
@@ -150,6 +150,68 @@ def option(
     ...
 
 
+@overload
+def option(
+    *param_decls: str,
+    cls: Type[Option] = ...,
+    # Option
+    show_default: bool = ...,
+    prompt: Union[bool, Text] = ...,
+    confirmation_prompt: bool = ...,
+    hide_input: bool = ...,
+    is_flag: Optional[bool] = ...,
+    flag_value: Optional[Any] = ...,
+    multiple: bool = ...,
+    count: bool = ...,
+    allow_from_autoenv: bool = ...,
+    type: Type[str] = ...,
+    help: Optional[str] = ...,
+    # Parameter
+    default: Optional[Any] = ...,
+    required: bool = ...,
+    callback: Callable[[Context, Union[Option, Parameter], str], Any] = ...,
+    nargs: Optional[int] = ...,
+    metavar: Optional[str] = ...,
+    expose_value: bool = ...,
+    is_eager: bool = ...,
+    envvar: Optional[Union[str, List[str]]] = ...,
+    # User-defined
+    **kwargs: Any,
+) -> Callable[[_F], _F]:
+    ...
+
+
+@overload
+def option(
+    *param_decls: str,
+    cls: Type[Option] = ...,
+    # Option
+    show_default: bool = ...,
+    prompt: Union[bool, Text] = ...,
+    confirmation_prompt: bool = ...,
+    hide_input: bool = ...,
+    is_flag: Optional[bool] = ...,
+    flag_value: Optional[Any] = ...,
+    multiple: bool = ...,
+    count: bool = ...,
+    allow_from_autoenv: bool = ...,
+    type: Type[int] = ...,
+    help: Optional[str] = ...,
+    # Parameter
+    default: Optional[Any] = ...,
+    required: bool = ...,
+    callback: Callable[[Context, Union[Option, Parameter], str], Any] = ...,
```

Comment:
Sorry, it seems that I didn't save my comment. I think the third argument to the callable should be `int` in this case, not `str`, if I understand the `option` API correctly.

Reply:
Ah.

Reply:
Updated.

Reply:
You updated it in both overloads, but the idea is that this argument type is equivalent to the `type` argument of the overload. So it should be `str` in the first overload and `int` in the second one.

Reply:
Ah. Fixing.